analyzer/src/main/resources/META-INF/services/org.ossreviewtoolkit.model.MapperCustomizer

```diff
@@ -0,0 +1 @@
+org.ossreviewtoolkit.analyzer.managers.utils.CustomAnalyzerResultSerializerModule
```

Comment:
Is this now still needed? I thought we got rid of the `ServiceLoader`-based approach in the new iteration.

Reply:
Well spotted, you are right. The file was removed.